azure-pipelines-perf.yml

```diff
@@ -1,3 +1,9 @@
+parameters:
+- name: isOfficial
```

Comment:
What is this parameter for?

Reply:
We've been using that to mark the unofficial pipelines like PR, perf, localization, richnav, basically all ymls used in the public dnceng that are not directly producing the build for publishing to be marked accordingly - https://github.com/dotnet/vscode-csharp/pull/6972/files#diff-7915b9b726a397ae7ba6af7b9703633d21c031ebf21682f3ee7e6a4ec52837a5:~:text=parameters%3A-,isOfficial%3A%20false,-%2D%20stage%3A

Reply:
Is this parameter used? anywhere?

Reply:
removed for now, it was meant to filter out the 1ES scan